framework/mesh/mesh_continuum/mesh_continuum.cc

```diff
@@ -1132,4 +1131,76 @@ MeshContinuum::GetGlobalNumberOfCells() const
   return num_cells;
 }
 
+void
+MeshContinuum::SetMatIDToAll(int mat_id)
```

Comment:
check with @zhardy-lanl about the final name for this one

Reply:
I like the `SetUniformMaterialID` better that this and I will add a patch for that.